### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by ProofStakePool**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+   {
+    "ticker": "PROOF",
+    "poolId": "pool17s9pcsnhgueer24p4lk53ruglr0r8009yxfrqvwgk5gx6r44cpe",
```
Comment (Bez625) [inline registry.json:3606]: This doesnt look like a real pool id.. I think it is actually `f40a1c4277473391aaa1afed488f88f8de33bde521923031c8b5106d` but please check and fix

File: registry.json [hunk 2]
```diff
@@ -3600,5 +3600,19 @@
...
+    "social": {
+      "website": "https://www.proofstakepool.com",
+      "twitter": "",
```
Comment (Bez625) [inline registry.json:3609]: Please add a social media presence of some kind

File: registry.json [hunk 3]
```diff
@@ -3600,5 +3600,19 @@
...
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```
Comment (Bez625) [inline registry.json:3614]: Please can you add a TG handle? This must be the handle of a user and not a group or channel.

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 2 above)
Comment (ProofStakePool) [reply registry.json:3609]: Twitter: ProofStakePool

File: registry.json (hunk 3 above)
Comment (ProofStakePool) [reply registry.json:3614]: Telegram: sergejmaehler

File: registry.json [hunk 4]
```diff
@@ -3657,6 +3657,19 @@
...
+    "social": {
+      "website": "https://www.proofstakepool.com",
+      "twitter": "ProofStakePool",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "sergejmaehler",
+    "operatorName": "Sergej Maehler",
+    "github": "ProofStakePool"
```
Comment (Bez625) [inline registry.json:3672]:
```suggestion
    "github": "ProofStakePool"
    },
```

**Pull request merged**